src/runtime_src/tools/xclbinutil/CMakeLists.txt

```diff
@@ -147,6 +147,9 @@ if(NOT WIN32)
   set(TEST_OPTIONS " --resource-dir ${CMAKE_CURRENT_SOURCE_DIR}/unittests/MCS")
```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
copyright date updated